core/src/main/java/com/cryptape/cita/protocol/system/CITAjSystemContract.java

```diff
@@ -604,45 +678,33 @@ public QueryResourceResult queryResource(String fromAddr, String permissionAddr)
     /**
      * new group
      * @see <a href="https://docs.citahub.com/zh-CN/cita/sys-contract-interface/interface#newgroup">newGroup</a>
-     * @param superAdminAddress the address of super_admin
      * @param groupName the name of group to be created
      * @param accounts accounts added to the group
-     * @param adminPrivateKey the private key of super_admin
+     * @param adminPrivatekey the private key of super_admin
      * @param version
      * @param chainId
      * @return the transaction hash for creating group
      * @throws IOException
      */
-    public String newGroup(String superAdminAddress,
-                           String groupName,
-                           List<String> accounts,
-                           String adminPrivateKey,
-                           int version,
-                           BigInteger chainId) throws IOException {
-        // account addresses convert to Address object list
-        List<Address> addresses = new ArrayList<>(accounts.size());
-        for(String acc : accounts){
-            addresses.add(new Address(acc));
-        }
-
-        // groupName string convert to bytes32
+    public String newGroup(String groupName, List<String> accounts, String adminPrivatekey, int version, BigInteger chainId) throws Exception {
         String nameHex = Util.addUpTo64Hex(ConvertStrByte.stringToHexString(groupName));
         byte[] nameBytes = ConvertStrByte.hexStringToBytes(Numeric.cleanHexPrefix(nameHex));
-
-        // build input parameters
+        List<Address> addrsToAdd = new ArrayList<>();
```

Comment:
I think `addrsToAdd` is uncomfortable, not simply. And I think variable names should tell other developers what it is not how to do.

Reply:
So, what's your opinion? I found 'addrsToAdd' was used by many places.

Reply:
`addrsToAdd` is OK for now, we should keep the same naming style. We can refactor them together later when we have a better naming style.

Reply:
OK